Sources/MixedReality/HoloLensCapture/HoloLensCaptureExporter/DataExporter.cs

```diff
@@ -116,21 +117,29 @@ void VerifyMutualExclusivity(dynamic s0, dynamic s1)
 
                 if (encodedImageCameraView != null)
                 {
-                    // export encoded camera view as is
                     VerifyMutualExclusivity(imageCameraView, gzipImageCameraView);
-                    encodedImageCameraView.Export(name, exportCommand.OutputPath, streamWritersToClose);
+                    if (isNV12)
+                    {
+                        // export NV12-encoded camera view as lossless PNG
+                        encodedImageCameraView?.Decode(decoder)?.Encode(pngEncoder).Export(name, exportCommand.OutputPath, streamWritersToClose);
```

Comment:
Delivery policies? We need to make sure we're not dropping messages.

Reply:
The whole pipeline is created with `DeliveryPolicy.SynchronousOrThrottle`.

Reply:
I think we want `DeliveryPolicy.Unlimited` on the pipeline, right? `SynchronousOrThrottle` might sometimes drop messages if the synchronous attempt fails and the component cannot be throttled? Or maybe not, but still, seems more straightforward to just specify `Unlimited`.